runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java

```diff
@@ -102,10 +114,33 @@ public void onScriptCallbackEvent(ScriptCallbackEvent e)
 						continue;
 					}
 
-					level += Experience.getLevelForXp(client.getSkillExperience(s));
+					if (config.alterTotalLevel())
```

Comment:
What is this code doing? The proper behavior for when this config is disabled is to not change the script behavior, not to do some other code.